src/Config/RectorConfig.php

```diff
@@ -212,6 +212,21 @@ public function rule(string $rectorClass): void
 
         // for cache invalidation in case of change
         SimpleParameterProvider::addParameter(Option::REGISTERED_RECTOR_RULES, $rectorClass);
+
+        // for cache invalidation in case of change
+        SimpleParameterProvider::addParameter(Option::REGISTERED_RECTOR_RULES, $rectorClass);
```

Comment:
this double call with above line 214

Reply:
Fixed :+1:

Reply:
https://github.com/rectorphp/rector-src/pull/5607